### DISCUSSION TITLE
Change elements of the reference style to another language

### DISCUSSION THREAD

**Discussion by yello-data** [quarto-dev/quarto-cli#3823] [Q&A] [answered]
Hi,
I'm using Quarto Books and by default the reference style is in English. For instance, for several authors it uses "and"  (e.g. Wickham **and** Bryan 2022).
I would like to change the elements to another language (i.e. to convert "and" to "y" in Spanish or "i" in Catalan). 
Is there any way to make these changes in the _quarto.yml document?
Thank you in advance,

**Comment by mcanouil** [comment, marked as answer]
References are controlled by CSL.
You need to find and use one that suits you, and.or modify one.

See <https://quarto.org/docs/authoring/footnotes-and-citations.html#citation-style>.
